scenes/globals/property_utils.gd

```diff
@@ -40,7 +40,7 @@ static func expose_children_property(
 		var property_dict: Dictionary = get_property_dict(child, property)
 		if not property_dict.is_empty():
 			property_dict = property_dict.duplicate()
-			property_dict["name"] = "%s__%s" % [child.name.to_snake_case(), property]
+			property_dict["name"] = "%s__%s" % [child.name, property]
```

Comment:
Does this break any properties currently saved in the game? I think yes:

```
scenes/game_elements/props/buildings/house/house_1.tscn:11:house_sprite__texture = ExtResource("2_qufpk")
scenes/game_elements/props/buildings/house/house_2.tscn:11:house_sprite__texture = ExtResource("2_1mgcp")
scenes/game_elements/props/decoration/bush/bush.tscn:8:bush_sprite__texture = ExtResource("1_p46yk")
scenes/game_elements/props/decoration/spool/spool.tscn:11:spool_sprite__texture = ExtResource("2_dh24t")
...
```

Reply:
You are right. Opening those scenes again corrected it. Then, for the scenes that included these decorations, I went over them and corrected the property manually in https://github.com/endlessm/threadbare/pull/398/commits/5bd21a60426af0ac26f24bb73cb4a976a3b71c46